scipy/sparse/tests/test_coo.py

```diff
@@ -268,6 +541,20 @@ def test_2d_matmul_multivector():
     assert_equal(res.toarray(), exp)
 
 
+@pytest.mark.parametrize(('mat_shape', 'vec_shape'), [((2,3,4,5), (5,)), ((0,0), (0,)),
+                                            ((2,3,4,7,8), (8,)), ((4,4,2,0), (0,)),
+                                            ((6,5,3,2,4), (4,1)), ((2,5), (5,1)), ((3,), (3,1)),])
```

Comment:
These are hard to read, near the end of the line limit (88 chars), and not uniformly indented which would make it worse. Perhaps the most readable aproach is to create a variable on the previous line to hold all the cases.
```python
mat_vec_shapes = [
    ((2, 3, 4, 5), (5,)), 
    ((0, 0), (0,)), 
    ((2, 3, 4, 7, 8), (8,)),
    ((4, 4, 2, 0), (0,)),
    ((6, 5, 3, 2, 4), (4, 1)), 
    ((2, 5), (5, 1)),
    ((3,), (3, 1)),
]
```
followed by the parametrize decorator.  That will help the `lint` CI test to pass.

Reply:
Thanks - fixed at c3b9594.